src/server/routes/index.ts

```diff
@@ -1,58 +1,35 @@
 import { FastifyInstance } from "fastify";
-import { getContractExtensions } from "./contract/metadata/extensions";
-import { readContract } from "./contract/read/read";
-import { writeToContract } from "./contract/write/write";
-
-// Transactions
-import { cancelTransaction } from "./transaction/cancel";
-import { getAllTx } from "./transaction/getAll";
-import { getAllDeployedContracts } from "./transaction/getAllDeployedContracts";
-import { retryTransaction } from "./transaction/retry";
-import { checkTxStatus } from "./transaction/status";
-
-// Extensions
-import { erc1155Routes } from "./contract/extensions/erc1155";
-import { erc20Routes } from "./contract/extensions/erc20/index";
-import { erc721Routes } from "./contract/extensions/erc721";
-import { marketplaceV3Routes } from "./contract/extensions/marketplaceV3/index";
-import { prebuiltsRoutes } from "./deploy";
-
-// Chains
-import { getChainData } from "./chain/get";
-import { getAllChainData } from "./chain/getAll";
-
-// Contract events
-import { getAllEvents } from "./contract/events/getAllEvents";
-import { getEvents } from "./contract/events/getEvents";
-
-// Contract roles
-import { getRoles } from "./contract/roles/read/get";
-import { getAllRoles } from "./contract/roles/read/getAll";
-import { grantRole } from "./contract/roles/write/grant";
-import { revokeRole } from "./contract/roles/write/revoke";
-
-// Contract royalties
-import { getDefaultRoyaltyInfo } from "./contract/royalties/read/getDefaultRoyaltyInfo";
-import { getTokenRoyaltyInfo } from "./contract/royalties/read/getTokenRoyaltyInfo";
-import { setDefaultRoyaltyInfo } from "./contract/royalties/write/setDefaultRoyaltyInfo";
-import { setTokenRoyaltyInfo } from "./contract/royalties/write/setTokenRoyaltyInfo";
-
-// Contract metadata
-import { getABI } from "./contract/metadata/abi";
-import { extractEvents } from "./contract/metadata/events";
-import { extractFunctions } from "./contract/metadata/functions";
-
-// Wallet
+import { createAccessToken } from "./auth/access-tokens/create";
```

Comment:
I'm unsorting the imports here. I believe all devs should be using auto-imports from our IDE which will just add/remove it as needed. It's not smart enough to put imports in the right section, nor do we really care.

Reply:
we should be auto generating barrel files using something like [barrelsby](https://www.npmjs.com/package/barrelsby)